transmart-rest-api/grails-app/services/org/transmartproject/rest/dataExport/ExportAsyncJobService.groovy

```diff
@@ -0,0 +1,105 @@
+package org.transmartproject.rest.dataExport
+
+import com.recomdata.transmart.domain.i2b2.AsyncJob
+import grails.transaction.Transactional
+import org.grails.web.json.JSONObject
+import org.quartz.JobKey
+import org.quartz.Scheduler
+import org.transmartproject.db.multidimquery.query.InvalidQueryException
+import org.transmartproject.db.user.User
+
+@Transactional
+class ExportAsyncJobService {
+
+    Scheduler quartzScheduler
+
+    static String jobType = 'DataExport'
+
+    /**
+     * Get the list of jobs to show in the jobs tab
+     */
+    List<AsyncJob> getJobList(User user) {
+        // TODO add time constraints for retrived jobs
```

Comment:
For now we agreed that all the jobs will be received, later we should add some time limit (configurable) parameter